packages/main/cypress/specs/Link.cy.tsx

```diff
@@ -1,31 +1,278 @@
 import Link from "../../src/Link.js";
+import type { LinkClickEventDetail } from "../../src/Link.js";
+
+describe("General API", () => {
+	it("render initially", () => {
+		cy.mount(<Link></Link>);
+
+		cy.get("[ui5-link]")
+			.shadow()
+			.find(".ui5-link-root")
+			.should("exist");
+	});
+
+	it("tests href attributes", () => {
+		const HREF_ATTRIBUTE = "https://www.sap.com/index.html";
+
+		cy.mount(<Link></Link>);
+
+		cy.get("[ui5-link]")
+			.shadow()
+			.find(".ui5-link-root")
+			.as("innerLink");
+
+		cy.get("@innerLink")
+			.should("not.have.attr", "href");
+
+		cy.get("[ui5-link]").invoke("attr", "href", HREF_ATTRIBUTE);
+
+		cy.get("@innerLink")
+			.should("have.attr", "href", HREF_ATTRIBUTE);
+	});
+
+	it("tests rel attribute", () => {
+		cy.mount(<Link href="https://www.sap.com" target="_blank"></Link>);
+
+		cy.get("[ui5-link]")
+			.shadow()
+			.find(".ui5-link-root")
+			.should("have.attr", "rel", "noreferrer noopener");
+	});
+
+	it("tests target attributes", () => {
+		const TARGET_ATTRIBUTE = "_blank";
+
+		cy.mount(<Link></Link>);
+
+		cy.get("[ui5-link]")
+			.shadow()
+			.find(".ui5-link-root")
+			.as("innerLink");
+
+		cy.get("@innerLink")
+			.should("not.have.attr", "target");
+
+		cy.get("[ui5-link]").invoke("attr", "target", TARGET_ATTRIBUTE);
+
+		cy.get("@innerLink")
+			.should("have.attr", "target", TARGET_ATTRIBUTE);
+	});
```

Comment:
do we really need to test if we pass an attribute to the inner link?

Reply:
Will remove 2 ot these 3 tests, middle one is important.